guides/nestor/ISPsetup.md

```diff
@@ -0,0 +1,308 @@
+---
+Layout: post
+Title: ISP setup ubuntu
+Author: Nestor N Torres
+---
+
+### Contact
+- Slack: @nanjuan on wcscusf.slack.com
+- [LinkedIn](https://www.linkedin.com/in/nestor-n-torres-737172a5/)
+- Email: nestor@nntorres.com
+
+### Table
+1. [Prerequisites](#id-link-to-section)
+2. [Summary](#Summary)
+3. [First logical group of Page](#First)
+4. [Second logical group of Page](#Second)
+5. [Third Logical group of pare](#Third)
+
+## Prerequisites <a id="id-link-to-section"></a>
+1. If you have not setup your network adapter look here https://goo.gl/vs4cHC.
+2. Just copy the Prerequisites of peoples guides before yours here.
+3. Have your [virtual environment](link-to-guide) configured
+4. Have the [ISP](link-to-guide) gateway running.
+5. Have [pfSense](link-to-guide) running.
+
+## Summary <a id="Summary"></a>
+This tutorial will show you how to setup the ISP server you will need for your lab.
+
+<br>
+
+| Network adapters | Description                     |
+|------------------|---------------------------------|
+| Adapter 1        | Leave it to Nat                 |
+| Adapter 2        | Change to Host-only (vboxnet2)  |
+
+##### Supporting detail 
+The logical ideal of this tutorial is for you to be able to setup the ISP server for your lab. The ISP lab is the one that allow you to connect your other lab to the internet think at it as your internet provider at home.  
+
+## First logical group of Page <a id="First"></a>
+
+
+- Go to https://www.ubuntu.com/download/server and download the LTS server version. 
+
+1. Install Ubuntu Server in Virtual Machine
+2. On Virtual Box go to top left corner to New to create the virtual box.
+
+![Section2](https://i.imgur.com/QnoKC2F.png) 
+
+3. Create the name of this Box recommended name (ISP Ubuntu)
+- `In Type choose Linux if is not auto setected`
```

Comment:
This applies to any code comment you have like `In Type choose Linux if is not auto setected` and `Leave the Hard disk file type to (VDI)` and you have many more. Only use code comments for actual buttons/code. So number 2 says select new. It should say select `New`. Don't code the entire line.